cypress/fixtures/moderation_dashboard/mockedSubmissionResponse.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
let's move this to our pre-existing  `mockData` folder so we can keep it all together. The term `Fixtures` while often used, many people  don't understand what it means compared to mock data/fake data/etc.